### PR TITLE
GCode auto deletion

### PR DESCRIPTION
**Pull request by irlaec**

### CODE REVIEW

File: octoprint_mrbeam/__init__.py [hunk 1]
```diff
@@ -552,6 +554,15 @@ def on_settings_save(self, data):
                 )
             if "focusReminder" in data:
                 self._settings.set_boolean(["focusReminder"], data["focusReminder"])
+            if "gcodeAutoDeletion" in data:
+                self._settings.set_boolean(
+                    ["gcodeAutoDeletion"], data["gcodeAutoDeletion"]
+                )
```
Comment (amrsoll) [inline octoprint_mrbeam/__init__.py:563]: Is this necessary now? I thought `octoprint.plugin.SettingsPlugin.on_settings_save(self, data)` now took care of that.

**Review by amrsoll — APPROVED** [review]
I'm happy to see such a compact front end code :smile:

**Comment by irlaec** [comment]
[Auto-delete Gcodes](https://trello.com/c/zGAUUQSM/652-auto-delete-gcodes)

**Comment by amrsoll** [comment]
[Do the Settings Properly (from now on)](https://trello.com/c/Kk4ABsac/677-do-the-settings-properly-from-now-on)

**Pull request merged**